sycl/source/detail/graph_impl.cpp

```diff
@@ -523,6 +526,11 @@ void exec_graph_impl::createCommandBuffers(sycl::device Device) {
 
   // TODO extract kernel bundle logic from enqueueImpKernel
   for (auto Node : MSchedule) {
+    // Empty nodes are node processed as other nodes, but only their
+    // dependencies are propagated in findRealDeps
```

Comment:
```suggestion
    // Empty nodes are not processed as other nodes, but only their
    // dependencies are propagated in findRealDeps
```
I think this correction is correct but maybe I'm reading it wrong.